### PR TITLE
fix: add missing calling button dimension

### PR DESCRIPTION
**Pull request by alexandreferris**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [X] contains a reference JIRA issue number like `SQPIT-764`
  - [X] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

[AR-1755 - Calling buttons with wrong dimension](https://wearezeta.atlassian.net/browse/AR-1755)

### Issues

There was one missing screen with wrong calling control button dimension.

### Solutions

Add correct dimension to missing screen.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-614/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3452267)(3452267)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3452267)(3452267)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-614/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3466859)(3466859)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3466859)(3466859)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/calling/initiating/InitiatingCallScreen.kt [hunk 1]
```diff
@@ -25,7 +24,7 @@ import androidx.hilt.navigation.compose.hiltViewModel
 import com.wire.android.R
 import com.wire.android.ui.calling.ConversationName
 import com.wire.android.ui.calling.controlButtons.CallOptionsControls
-import com.wire.android.ui.calling.controlButtons.HangUpButton
+import com.wire.android.ui.calling.controlButtons.DeclineButton
```
[the remark this answers is not in the document]
Comment (ohassine) [reply app/src/main/kotlin/com/wire/android/ui/calling/initiating/InitiatingCallScreen.kt:27]: yes let's do it

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-614/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3467812)(3467812)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3467812)(3467812)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-614/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3467933)(3467933)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3467933)(3467933)-dev-release.apk)

**Review by ohassine — APPROVED** [review]
well done ✅

**Pull request merged**